tests/wakunode_rest/test_rest_health.nim

```diff
@@ -0,0 +1,77 @@
+{.used.}
+
+import
+  std/tempfiles,
+  stew/shims/net,
+  testutils/unittests,
+  presto,
+  presto/client as presto_client,
+  libp2p/peerinfo,
+  libp2p/multiaddress,
+  libp2p/crypto/crypto
+import
+  ../../waku/waku_node,
+  ../../waku/node/waku_node as waku_node2,  # TODO: Remove after moving `git_version` to the app code.
+  ../../waku/node/rest/server,
+  ../../waku/node/rest/client,
+  ../../waku/node/rest/responses,
+  ../../waku/node/rest/health/handlers as health_api,
+  ../../waku/node/rest/health/client as health_api_client,
+  ../../waku/waku_rln_relay,
+  ../testlib/common,
+  ../testlib/wakucore,
+  ../testlib/wakunode
+
+
+proc testWakuNode(): WakuNode =
+  let
+    privkey = crypto.PrivateKey.random(Secp256k1, rng[]).tryGet()
+    bindIp = ValidIpAddress.init("0.0.0.0")
+    extIp = ValidIpAddress.init("127.0.0.1")
+    port = Port(58000)
```

Comment:
Let's not hardcode ports unless necessary (JSON-RPC or REST at the moment) - please use `Port(0)`

Reply:
Fixed.